docs/README.md

```diff
@@ -1,12 +1,25 @@
-# Scylla documentation
+# ScyllaDB Documentation
 
-This is the repository for [Scylla documentation site](http://docs.scylladb.com/)
+This repository contains the source files for ScyllaDB Open Source documentation.
 
-To report an issue with the documentation, please use GitHub issues.
+- The `dev` folder contains developer-oriented documentation related to the ScyllaDB code base. It is not published and is only available via GitHub.
+- All other folders and files contain user-oriented documentation related to ScyllaDB Open Source and are sources for [docs.scylladb.com](https://docs.scylladb.com/).
 
-This repository accepts GitHub pull requests.
+To report a documentation bug or suggest an improvement, open an issue in [GitHub issues](https://github.com/scylladb/scylla/issues) for this project.
 
-**Send pull requests to the master branch, not gh-pages.  gh-pages will be overwritten by deploy without warning.**
+To contribute to the documentation, open a GitHub pull request.
```

Comment:
It seems this line simply repeats the line before?

Reply:
@nyh I'm not sure which line you're referring to. The previous line explains where issues should be reported. This one talks about contributing to the docs.

Reply:
Ok, I see what you mean - you have the option to report an issue, and a different option to fix the issue yourself and suggest a patch. I guess that indeed it's not the same thing.